backend/src/services/integrationService.ts

```diff
@@ -688,14 +688,17 @@ export default class IntegrationService {
       await MemberAttributeSettingsRepository.findAndCountAll({}, this.options)
     ).rows
 
-    const identities = await TenantRepository.getAvailablePlatforms(tenantId, this.options)
+    const platforms = (await TenantRepository.getAvailablePlatforms(tenantId, this.options)).map( (p) => p.platform)
+
+    const hubspotId = integration.settings.hubsputId
```

Comment:
Typo?
```
const hubspotId = integration.settings.hubsputId
                                            ^
```

Reply:
yeah fixed - thanks for spotting it :pray: